### PR TITLE
misc edits

### PR DESCRIPTION
**Pull request by smichr**
#### Release Notes

NO ENTRY

### CODE REVIEW

File: sympy/geometry/tests/test_line.py [hunk 1]
```diff
@@ -796,11 +796,11 @@ def test_ray_generation():
...
 
 
-def test_symbolic_intersect():
-    # Issue 7814.
+def test_issue_7814():
     circle = Circle(Point(x, 0), y)
     line = Line(Point(k, z), slope=0)
-    assert line.intersection(circle) == [Point(x + sqrt((y - z) * (y + z)), z), Point(x - sqrt((y - z) * (y + z)), z)]
+    assert line.intersection(circle) == [
+        Point2D(x + sqrt(y**2 - z**2), z), Point2D(x - sqrt(y**2 - z**2), z)]
```
Comment (smichr) [inline sympy/geometry/tests/test_line.py:803]:
```suggestion
    assert line.intersection(circle) == [
        Point(x + sqrt((y - z)*(y + z)), z), Point(x - sqrt((y - z)*(y + z)), z)]
```

**Comment by smichr** [comment]
blocked by #23840

File: sympy/geometry/tests/test_line.py (hunk 1 above)
Comment (smichr) [inline sympy/geometry/tests/test_line.py:803]:
```suggestion
    _s = sqrt((y - z)*(y + z))
    assert line.intersection(circle) == [Point2D(x + _s, z), Point2D(x - _s, z)]
```

**Comment by oscargus** [comment]
Not sure if you plan to sort out the XXX TODO in this PR. If not, feel free to merge.

**Comment by smichr** [comment]
> sort out the XXX TODO

Thanks for checking. I must have lost that edit in the shuffle. I will re-apply the change.

**Pull request merged**